update README file about new release process (#33)
tradebyte/paci: README.md

5. Check if everything worked on the test.pypi.org site.
6. Publish it for real: `twine upload dist/*`

> ### Releasingi automatically

Reviewer: ```suggestion
### Releasing to PyPI automatically
```

> You can release a new version by workflows and automatically.

Reviewer: ```suggestion
You can release a new version by using Github workflow automatically.
```

1. Bump up the version number in `paci/__init__.py` according to [semver](https://semver.org/).

> 2. Create a new release tag version and it will start our pipeline and push the new version to PyPI and PyPI test.

Reviewer: ```suggestion
2. Create a new release tag version and it will start our pipeline and push the new version to PyPI and PyPI test after it is merged to main.
```
